pyxem/utils/segment_utils.py

```diff
@@ -145,12 +145,14 @@ def separate_watershed(
     # Find the coordinates of the local maxima of the distance transform.
     local_maxi = peak_local_max(
         distance,
-        indices=False,
         min_distance=1,
         num_peaks=max_number_of_grains,
         exclude_border=exclude_border,
         threshold_rel=None,
     )
+    peak_mask = np.zeros_like(distance, dtype=bool)
```

Comment:
Again, does this work for scikit-image >= 0.18.0? Or should the minimal version be increased to 0.19.0?

We should have a runner with tests using minimal supported versions...

Reply:
Sounds like a good idea.  I'll add that as well.  You are correct that the minimum version should be 0.19.0